PatelDLTGroupStamps/Models/StampStore.swift

```diff
@@ -0,0 +1,76 @@
+  var id = UUID()
+  var seller: String
+  var name: String? = nil
+  let description: String
+  var image: String
+  let price: String
+  var discountTypes: [String] = []
+}
+
+var unitedNationsStamps = [
+  UNStamp(seller: "hiplatics", name: "Vienna collector stamp", description: "UNESCO World Heritage Site I", image: "image1", price: "2.99"),
+  UNStamp(seller: "hiplatics", name: "Vienna collector stamp", description: "UNESCO World Heritage Site II", image: "image2", price: "2.99"),
+  UNStamp(seller: "hiplatics", name: "Vienna collector stamp", description: "Alhambra Granada Spain World Heritage", image: "image3", price: "2.19"),
+  UNStamp(seller: "hiplatics", name: "Geneva collector stamp", description: "UNESCO World Heritage Site Germany", image: "image4", price: "6.99"),
+  UNStamp(seller: "hiplatics", name: "Geneva collector stamp", description: "Adelie Penguin Pygoscelis", image: "image5", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "Geneva collector stamp", description: "UNESCO World Heritage Site Italy", image: "image6", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "United States collector stamp", description: "Distinguished Soldier 2nd Lt Audie Baby Murphy", image: "image7", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "Monaco collector stamp", description: "Place DE Lagare DE Monaco Vers", image: "image8", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "Vatican City collector stamp", description: "Vatican City", image: "image9", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "Geneva collector stamp", description: "UN Geneva", image: "image10", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "Israel collector stamp", description: "Tete-Beche Pair", image: "image11", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "French Morocco collector stamp", description: "Ben Smine Sanatorium", image: "image12", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "UN New York collector stamp", description: "65th Anniversary Of The United Nations", image: "image13", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "France collector stamp", description: "UNESCO World Heritage Medina Siam", image: "image14", price: "3.99"),
+  UNStamp(seller: "hiplatics", name: "UN Geneva collector stamp", description: "St. Catherine Monastery", image: "image15", price: "3.99")
+]
+
+// MARK: URL - Using FileManager
+let stampsPListURL = URL(fileURLWithPath: "Stamp", relativeTo: FileManager.documentsDirectoryURL.appendingPathExtension("plist"))
+
+// MARK: Assignment 1: Saving data to Plist only
+private func savePListStamps() {
+  let encoder = PropertyListEncoder()
+  encoder.outputFormat = .xml
+  
+  do {
+    let encodedData = try encoder.encode(unitedNationsStamps)
+    try! encodedData.write(to: stampsPListURL, options: .atomicWrite)
+    print(encodedData)
+  } catch let error {
+    print(error)
+  }
+}
+// MARK: Assignment 2: Loading the saved data
+private func loadPListStamps() {
```

Comment:
Therefore, there are no files you can load.

Reply:
Thank You, can you assist so that I can re-submit?

Let me see what I can do on second attempt.

-pravin

Reply:
Hi, I have updated the project app.

 Created a new StampStore and StampPListView swift file also incorporated a new struct along with saving and loading data but not the preferred way to do things.

  Not sure if the the output is correct or not by running the app.

  Please note I have updated the Hiplatelics.swift to start the view on StampPListView.